addon/mixins/scroller-events.js

```diff
@@ -78,11 +78,20 @@ function handleMove(e) {
   }
 }
 
+function scrollingDidStart() {
+  console.log('scrollingDidStart');
+  Ember.run(() => {
+    this.get('scrollerRegistry').startScrolling();
+  });
+}
+
 function scrollingDidComplete() {
+  console.log('scrollingDidComplete');
   Ember.run(() => {
     this.get('scrollerRegistry').endScrolling();
   });
   setTimeout(()=>{
+    // if (this.isDestroyed || this.isDestroying) { return; }
     Ember.run(() => {
       this.set('isScrolling', false);
```

Comment:
Occasionally we will see errors in the console about trying to call set on a destroyed object. Wondering if we should check if destroyed first, like the commented out line above...